src/Renderer/Shader/ProjectiveTextureFS.glsl

```diff
@@ -0,0 +1,20 @@
+precision highp float;
```

Comment:
there's no `include` to add `precision highp float;`?

Reply:
I didn't find that